eng/pipelines/runtimelab.yml

```diff
@@ -0,0 +1,74 @@
+# Setting batch to true, triggers one build at a time.
+# if there is a push while a build in progress, it will wait,
+# until the running build finishes, and produce a build with all the changes
+# that happened during the last build.
+trigger:
+  batch: true
+  branches:
+    include:
+    - feature/*
```

Comment:
Technically you don't need this one right?

Reply:
Technically not, but I wanted to leave it like this so that people have to do the minimal set of changes when starting a new experiment.